docs/install.md

```diff
@@ -5,8 +5,14 @@
 - PyTorch 1.3+
 - CUDA 9.2+ (If you build PyTorch from source, CUDA 9.0 is also compatible)
 - GCC 5+
-- [MMCV](https://mmcv.readthedocs.io/en/latest/#installation) 1.3.4
-- [MMDetection](https://mmdetection.readthedocs.io/en/latest/#installation) 2.12.0
+- [MMCV](https://mmcv.readthedocs.io/en/latest/#installation)
+- [MMDetection](https://mmdetection.readthedocs.io/en/latest/#installation)
+
+The compatible MMTracking, MMCV, and MMDetection versions are as below. Please install the correct version to avoid installation issues.
+
+|  MMTracking version |       MMCV version       |  MMDetection version |
+|:-------------------:|:------------------------:|:--------------------:|
+|        master       | mmcv-full>=1.3.3, <1.4.0 |  MMDetection=2.13.0  |
```

Comment:
Mark the version of mmtracking with 0.5.2.

Reply:
done